### DISCUSSION TITLE
Is it possible to change session order?

### DISCUSSION THREAD

**Discussion by falouu** [termux/termux-app#2736] [Ideas]
We can use a shortcut to go to next/previous session. It would be more useful and convenient, if we could also change sessions order (rearrange them).

Is this possible?

**Comment by agnostic-apollo** [comment]
https://wiki.termux.com/wiki/Terminal_Settings#Handling_terminal_sessions

**Reply by falouu** [reply]
I know about that, but I don't see how to change sessions order in that documentation. Am I missing something?

**Reply by agnostic-apollo** [reply]
That is not supported currently. Apologies for misreading.
